source/tests/CMakeLists.txt

```diff
@@ -1,13 +1,33 @@
 
+#
+# Configure CMake own project
+#
+
+cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../../cmake")
+
+include(${CMAKE_CURRENT_SOURCE_DIR}/../../cmake/Custom.cmake)
```

Comment:
Partially intended as I don't see any other quick solution.
There are three parts that are included from the main project: Custom.cmake, CompileOptions.cmake, and additional find scripts.